app/ide-desktop/lib/content/esbuild-config.ts

```diff
@@ -122,6 +123,28 @@ export function bundlerOptions(args: Arguments): esbuild.BuildOptions {
             // We do not control naming of third-party options.
             esbuildPluginAlias({ ensogl_app: ensoglAppPath }),
             esbuildPluginTime(),
+            // This must run before the copy plugin so that the generated `tailwind.css` is used.
+            {
+                name: 'enso-generate-tailwind',
+                setup: build => {
+                    build.onStart(() => {
+                        const source = path.resolve(THIS_PATH, 'src', 'tailwind.css')
+                        const dest = path.join(outputPath, 'tailwind.css')
+                        const config = path.resolve(THIS_PATH, 'tailwind.config.ts')
+                        console.log(`Generating tailwind css from ${source} to ${dest}`)
+                        childProcess.spawnSync(`node`, [
+                            '../../node_modules/.bin/tailwindcss',
+                            '-i',
+                            source,
+                            '-o',
+                            dest,
+                            '-c',
+                            config,
+                            '--minify',
+                        ])
```

Comment:
Please refactor these to variable / variables to make the expressions one-liners instead (see styling code guide for more info).

Reply:
what exactly should i refactor to variables?